system-tests/test/testConfigOverrides_spec.ts

```diff
@@ -33,6 +33,23 @@ describe('testConfigOverrides', () => {
     },
   })
 
+  // FIXME: bug in runner causes browser to hang in run mode when test:before:run throws an exception
```

Comment:
Nice, the tests throughout this PR look very thorough :100:

Reply:
Is there an issue for following up on this?

Reply:
I think this one https://github.com/cypress-io/cypress/issues/23039